### PR TITLE
Sb subject view

### PR DESCRIPTION
**Pull request by sbagg**
This is referring to issue Identify subject for "view all X" items accessed via subject pages #294.  Completed and in need of review.

### CODE REVIEW

File: client/public/elements/pages/concept/rp-page-concept.js [hunk 1]
```diff
@@ -205,8 +206,8 @@ export default class RpPageConcept extends RpUtilsLanding {
    */
   async _doResearcherQuery(id) {
     let data = await this.SubjectModel.getResearchers(id);
-
     this.tempResearch = data.payload.results;
+    console.log(this.tempResearch );
```
Comment (qjhart) [inline client/public/elements/pages/concept/rp-page-concept.js:210]: comment out

File: client/public/src/models/CollectionModel.js [hunk 2]
```diff
@@ -253,6 +261,8 @@ class CollectionModel extends BaseModel {
       return subFacets;
     }
 
+    console.log("MainFacet", payload);
```
Comment (qjhart) [inline client/public/src/models/CollectionModel.js:264]: Comment out

File: client/public/src/models/CollectionModel.js [hunk 3]
```diff
@@ -299,6 +309,8 @@ class CollectionModel extends BaseModel {
         text: `All People (${dataTotal})`, 
         href: this.constructUrl(elementQuery, urlParamsToIgnore)
       });
+      console.log("subFacets:", subFacets);
```
Comment (qjhart) [inline client/public/src/models/CollectionModel.js:312]: Comment Out

File: client/public/src/models/CollectionModel.js [hunk 4]
```diff
@@ -312,6 +324,7 @@ class CollectionModel extends BaseModel {
         text: `All Works (${dataTotal})`, 
         href: this.constructUrl(elementQuery, urlParamsToIgnore)
       });
+      console.log("subFacets:", subFacets);
```
Comment (qjhart) [inline client/public/src/models/CollectionModel.js:327]: Comment Out

File: client/public/elements/pages/concept/rp-page-concept.tpl.js [hunk 5]
```diff
@@ -213,14 +213,29 @@ return html`
...
-            <rp-person-preview
-              .data="${researcher}"
-              text-width="${this.peopleWidth}"
-              show-subjects
-              class="my-3">
-            </rp-person-preview>
-          `)}   
+          ${this.tempResearch.map((researcher,v) => 
+            html`
+            ${v <= 10 ? html`
```
Comment (qjhart) [inline client/public/elements/pages/concept/rp-page-concept.tpl.js:218]: Should this be v<= 8 ?

**Review by qjhart — CHANGES_REQUESTED** [review]

**Review by qjhart — APPROVED** [review]

**Comment by jrmerz** [comment]
@sbagg @qjhart This PR added some suspect code: https://github.com/ucd-library/rp-ucd-client/blame/jm-ucdid-switch/client/public/elements/utils/rp-utils-collection.js#L274 ... can you please investigate, it's throwing a bug for me.  To start, data.id only has 1 __ in it.  Please add a little inline documentation as well.

**Pull request merged**